workflows/amr/run.wdl

```diff
@@ -283,12 +283,15 @@ task RunResultsPerSample {
             return([i for i in set_list if i == i])
 
         ontology = json.load(open("~{card_ontology}"))
-        def get_high_level_classes(gene_name):
-            if gene_name not in ontology:
-                return []
-            if 'highLevelDrugClasses' not in ontology[gene_name]:
-                return []
-            return ontology[gene_name]['highLevelDrugClasses']
+        def get_high_level_classes(drug_classes_union):
```

Comment:
what's the form of `drug_classes_union`? A set of strings like "rifamycin antibiotic; peptide antibiotic"? Wondering why it would have length > 1 ever

Reply:
It's type is `list[str]` but there is usually only one element in the list, a string containing drug classes separated by semicolons, e.g.:

<img width="1247" alt="Screenshot 2023-07-18 at 09 15 36" src="https://github.com/chanzuckerberg/czid-workflows/assets/24234461/e9c73f93-eac5-4043-aeba-1a216e9a01a0">

The variable is created by this line of code:
```python
dc = remove_na(set(sub_df['Drug Class_contig_amr']).union(set(sub_df['Drug Class_kma_amr'])))
```
This can result in two elements in the list if the two columns differ.

Reply:
I'm hoping that soon I can move the python code out to a separate file and upgrade the container's python version to >= 3.9 so type hinting is a lot easier to add.

Reply:
In the meantime that's probably a misleading variable name, I'll change it to something better